libs/ui/src/ui_strings/app_strings.tsx

```diff
@@ -642,6 +642,12 @@ export const appStrings = {
     </UiString>
   ),
 
+  noteBmdBallotBoxIsFull: () => (
+    <UiString uiStringKey="noteBmdBallotBoxIsFull">
+      A poll worker must empty the full ballot box.
+    </UiString>
```

Comment:
We need different strings than the existing "Replace Ballot Box" text because the ballot box is not replaced like in VxScan; rather, it's opened, emptied, and then closed again.